spec/controllers/queue_items_controller_spec.rb

```diff
@@ -59,10 +59,71 @@
     end
     
     it "redirects to sign in path for unauthenticted users" do
-        video = Fabricate(:video)
-        post :create, video_id: video.id
-        response.should redirect_to(sign_in_path)
+      video = Fabricate(:video)
+      post :create, video_id: video.id
+      response.should redirect_to(sign_in_path)
```

Comment:
You are using `expect` syntax almost everywhere. You should not mix it with `should` syntax. Keep 1 style, while `expect` style is newer and preferred nowadays.